snap/README.md

```diff
@@ -0,0 +1,59 @@
+# Snap Package for ASIMOV Platform
+
+This repository provides the Snap package for the [ASIMOV Platform]
+
+## **Prerequisites**
+
+- A Linux distribution with [Snap support](https://snapcraft.io/docs/installing-snapd).
+- Snapd installed and enabled.
+- `wget` (for downloading the `.snap` file).
+
+## **Download**
+
+The Snap package will be downloaded into the `asimov` directory inside the home folder (`~/asimov`).
+
+```bash
+mkdir -p ~/asimov && cd ~/asimov
+wget https://github.com/asimov-platform/asimov-packaging/releases/download/v25.0.0-dev.3/asimov-cli_25.0.0-dev.3_amd64.snap -O asimov-cli.snap
```

Comment:
Better to have these instructions refer to [the "latest" release](https://docs.github.com/en/repositories/releasing-projects-on-github/linking-to-releases#linking-to-the-latest-release) instead of a specific release

Reply:
@artob it is done